CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.22)
```

Comment:
Could you add a comment that explains that this CMakeList.txt files are not used for compiling this project, but they can be handy if you are integrating into an existing CMake project.